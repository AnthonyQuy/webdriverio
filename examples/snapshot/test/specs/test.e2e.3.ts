import { expect, browser, $ } from '@wdio/globals'

describe('My Login application', () => {
    it('should login with valid credentials', async () => {
        await browser.url('https://the-internet.herokuapp.com/login')

        await $('#username').setValue('tomsmith')
        await $('#password').setValue('SuperSecretPassword!')
        await $('button[type="submit"]').click()
        await expect($('#flash')).toBeDisplayed()

        const element = await $('#flash')
        const html = await element.getHTML()
        await expect(html).toMatchSnapshot()
    })
})
