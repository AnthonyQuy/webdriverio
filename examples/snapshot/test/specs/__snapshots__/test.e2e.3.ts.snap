// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`My Login application > should login with valid credentials 1`] = `
"<div data-alert=\\"\\" id=\\"flash\\" class=\\"flash success\\">
            You logged into a secure area!
            <a href=\\"#\\" class=\\"close\\">Ã—</a>
          </div>"
`;
